### ISSUE TITLE
One-way device handling

### ISSUE THREAD

**Issue by radiocolin** [staromeste/homebridge-http-advanced-accessory#16]
I added a line to create a television service, since the underlying framework has added support for it:
`case "Television": newService = new Service.Television(this.name); break;`

By stepping through debug, I'm able to figure out all the required properties as I open the device in HomeKit (and see where it crashes).

I'm hitting a wall in that the device I'm talking to is one-way stateless... it can receive button press commands but has no method of reporting a status back to me.

**Is there any way to map a static response for each of the required getWhatevers?**

Eg:
```
                "getActiveIdentifier":{
                    "url":null
                },
```
This sort-of works, but pumps out a lot of errors in the console, and tells HomeKit the device is unresponsive. I'd just like to always have a static "1" returned for this get event.

**Comment by staromeste** [comment]
No, but a workaround would be to call an echo service on the internet like postal-echo:
https://postman-echo.com/get?foo1=bar1&foo2=bar2

And map a static value to report or you can report the value you previously sent using the placeholders described in the documentation.



Guido
